rllib/algorithms/algorithm_config.py

```diff
@@ -36,6 +36,26 @@
 )
 from ray.tune.logger import Logger
 
+"""TODO(jungong, sven): in "offline_data" we can potentially unify all input types
```

Comment:
Thanks for moving this here to not clutter the docs :)

Reply:
yeah, I think we still have too many of those left. let's clean them one by one.